circleguard/utils.py

```diff
@@ -110,3 +110,24 @@ def span_to_list(span):
         else:
             ret.add(int(s))
     return ret
+
+def ur_to_cv_ur(ur, mods):
```

Comment:
how about `convert_ur()` ?

Reply:
that implies both directions, but that's actually good, I'll add an extra `to` argument which is either `"cv"` or `"ucv"` and make it `convert_ur`

Reply:
I feel like a string as an argument seems overly complex. I would actually prefer the current way over that´.

Reply:
unless we want to create a `ur` object (not necessarily a terrible idea, but an over engineered solution atm) or have a `ConversionMode` enum, a string argument is the best solution to providing both directions

Reply:
ye but I mean `ur_to_cv_ur` and `cv_ur_to_ur` would be simpler than a string argument imo.

Reply:
the code duplication there is so nasty :/ disagree